### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#177] [Esercizi - Viaggi/Piazza] [answered]
![Es 4](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/1d20d3e3-a731-4854-8fce-337ea11965f1)

**Comment by Elia-Belli** [comment, marked as answer]
$$S_3 = \\{ \begin{pmatrix} 1& 2 & 3\\
1& 2& 3 \end{pmatrix}, \begin{pmatrix} 1& 2 & 3\\
1& 3& 2 \end{pmatrix},\begin{pmatrix} 1& 2 & 3\\
2& 1& 3 \end{pmatrix},\begin{pmatrix} 1& 2 & 3\\
3& 2& 1 \end{pmatrix},\begin{pmatrix} 1& 2 & 3\\
2& 3& 1 \end{pmatrix},\begin{pmatrix} 1& 2 & 3\\
3& 1& 2 \end{pmatrix}\\}$$

Affinchè un gruppo sia commutativo $s \star s' = s' \star s, \forall s,s' \in G$, quindi ci basta dimostrare che ciò non valga per una coppia $f,f' \in S_3$.
Siano:

$$f = \begin{pmatrix} 1& 2 & 3\\
1& 3& 2 \end{pmatrix}, f' = \begin{pmatrix} 1& 2 & 3\\
2& 1& 3 \end{pmatrix}$$

$$f \circ f' = \begin{pmatrix} 1& 2 & 3\\
3& 1& 2 \end{pmatrix}, f'\circ f = \begin{pmatrix} 1& 2 & 3\\
2& 3& 1 \end{pmatrix}$$

Abbiamo mostrato che $f \circ f' \neq f' \circ f$ quindi $S_3$ non è un gruppo commutativo (infatti in genere la composizione di funzioni non è commutativa)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/b87e0173-ac0a-4f38-ab26-ab91e645b019)

**Comment by Elia-Belli** [comment]
@CiottoloMaggico piccolo errore, non vuoi dimostrare $f \circ g \neq g \circ f,  \forall f,g \in G$, ma che esista almeno una coppia $f,g$ che non soddisfi la condizione (che è quello che hai fatto).

**Comment by CiottoloMaggico** [comment]
si giusto, cambio il per ogni con esiste

**Comment by daromacarlo** [comment]
![Immagine 2023-12-05 215157](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/8bd991af-97d4-45c2-bfcd-e7c1bf757e50)
